src/studio/src/designer/backend/Controllers/LanguagesController.cs

```diff
@@ -47,10 +47,8 @@ public ActionResult<List<string>> GetLanguages(string org, string repo)
             List<string> languages = new List<string>(_languagesService.GetLanguages(org, repo, developer));
 
             languages.Sort();
```

Comment:
Minor, but this should be moved to the service class to prevent logic from leaking into the controller. That way all places where a service is used will get the same result.